### PR TITLE
Intial Testing

### PR DESCRIPTION
**Pull request by nanthony21**
Very excited to see my favorite microscopy software compatible with my favorite language!

These are just a few minor changes that I made in my initial attempts to get this working for me. I'm still having some issues though.

Changes include:
 - Move classes, functions, etc. used for communication over zmq out of `acquisition` to their own `core` module.
 - Rather than hanging indefinitely, Bridge will throw a Timeout exception with an explanation if it doesn't connect in .5 seconds.
 - Replace an instance of using a mutable list as a default argument for a method. This can cause unexpected behaviour: https://docs.python-guide.org/writing/gotchas/

### CODE REVIEW

**Comment by henrypinkard** [comment]
Looks great! Happy to merge these changes now. Were there more things you wanted to include also?

**Comment by henrypinkard** [comment]
Actually, does moving into core module change `from pycromanager.acquire import bridge` to `from pycromanager.core import bridge`? If so all documentation and examples will need to be updated

**Comment by nanthony21** [comment]
Ah yes, good point it will change the import paths. I'll fix it.

**Comment by nanthony21** [comment]
Looking through examples all of the imports are directly from the package, e.g. `from pycromanager import Bridge`, so we should be good to go.

I changed how things are imported into `__init__.py`. otherwise we end up with references to modules that are imported internally. It doesn't really matter but it's cleaner without them.

**Comment by nanthony21** [comment]
Hold on a sec, that last merge didn't work out properly

**Comment by nanthony21** [comment]
Ok, this PR is back up to date with master again. However the `view changes` button for your review isn't working. Which import are you asking about?

File: pycromanager/core.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from base64 import standard_b64encode, standard_b64decode
 import numpy as np
 import zmq
+from types import MethodType
```
Comment (henrypinkard) [inline pycromanager/core.py:8]: Don't understand the purpose of this

**Comment by henrypinkard** [comment]
Ready to merge?

**Comment by nanthony21** [comment]
Yes, ready.

**Comment by henrypinkard** [comment]
Thanks for the contribution!

**Pull request merged**